packages/http-client/src/ceramic-http-client.ts

```diff
@@ -125,6 +125,8 @@ export default class CeramicClient implements CeramicApi {
     const docIdStr = doc.id.toString()
     if (!this._docmap[docIdStr]) {
       this._docmap[docIdStr] = doc
+    } else {
+      this._docmap[docIdStr].state = doc.state
```

Comment:
what's the difference between setting the entire doc here vs just the state?

Reply:
If you have a pointer to this doc elsewhere you will get the updated version.

Reply:
Added the emitting of the event now (same as _syncState).

Reply:
hmm... _syncState only emits a change event if the state is actually different than the cached version.  Here we emit the event even if nothing changed

Reply:
Hm, you're right. I'll see if there is an easy fix.